tests/bud.bats

```diff
@@ -693,3 +693,49 @@ load helpers
   buildah rm ${cid}
   buildah rmi ${target}
 }
+
+@test "bud-onbuild" {
+  target=onbuild
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/onbuild
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild1" "RUN touch /onbuild2"]' ]
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  run ls ${root}/onbuild1 ${root}/onbuild2
+  echo "$output"
+  [ "$status" -ne 0 ]
+  buildah umount ${cid}
+  buildah rm ${cid}
+
+  target=onbuild-image2
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} -f Dockerfile1 ${TESTSDIR}/bud/onbuild
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild3"]' ]
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  run ls ${root}/onbuild1 ${root}/onbuild2
+  echo "$output"
+  [ "$status" -eq 0 ]
+  run ls ${root}/onbuild3
+  echo "$output"
+  [ "$status" -ne 0 ]
+  buildah umount ${cid}
+
+  target=onbuild-image3
+  run buildah --debug=false config --onbuild "RUN touch /onbuild4" ${cid}
```

Comment:
Think you need 
--signature-policy ${TESTSDIR}/policy.json 
here too.

Reply:
I think you only need signature-policy when doing a push or a pull.

Reply:
My bad, yep, not needed for config and it looks to be blowing up the tests.  You do need it for the commit a few lines down,  Sorry for the dyslexic moment.    bud, commit, from and push commands require it in the buds tests.